libraries/HumanPoseFuser.js

```diff
@@ -189,7 +190,7 @@ class HumanPoseFuser {
         /** max velocity of whole body - picked 2 m/s (average walking speed is 1.4 m/s) */
         this.maxHumanVelocity = 2.0;  // unit: mm/ms
         /** method for fusing poses for the same person from several views/toolbox apps. */
-        this.fusionMethod = 'BestSingleView'; // 'BestSingleViewWithMultiviewCorrection';
+        this.fusionMethod = 'BestSingleView'; // 'BestSingleViewWithCorrection'; // 'BestSingleViewWithMultiviewCorrection';
```

Comment:
Since you have FusionMethod available above, it would be more typo-redundant (and, depending on ide support, enable autocomplete) to write this as FusionMethod.BestSingleView instead of the string constant

Reply:
Right, good point.